docs/examples/general/data_loading/webdataset.ipynb

```diff
@@ -0,0 +1,514 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        random_current_state = random.getstate()
</pre><p>Maybe add random state context class that you can reuse later:</p><pre class="ql-syntax" spellcheck="false">class RandomStateContext(object):
    def __init__(self):
        self.prev_state = None

    def __enter__(self, context):
        self.prev_state = random.getstate()
        return self

    def __exit__(self ,type, value, traceback):
        random.setstate(self.prev_state)
</pre><p> </p><p>And later you can do:</p><pre class="ql-syntax" spellcheck="false">with RandomStateContext() as random_ctx:
    random.seed(...)
    while True:
        random.randint(...)

# Previous state will be restored
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>